.circleci/config.yml

```diff
@@ -794,7 +794,7 @@ jobs:
                 --fs-no-reconall --use-syn-sdc --ignore slicetiming \
                 --dummy-scans 1 --sloppy --write-graph \
                 --output-spaces MNI152NLin2009cAsym \
-                --mem-mb 14336 --nthreads 4 -vv
+                --mem-mb 14336 --nthreads 4 -vv --notrack
```

Comment:
How much effort would it be to split debug/production pings for migas? It seems useful that we cross-compare with sentry on these debug heartbeats.